nidirect_landing_pages/nidirect_landing_pages.routing.yml

```diff
@@ -0,0 +1,8 @@
+
+nidirect_landing_pages.landing_pages_choose_block_controller_build:
+  path: '/nidirect_landing_pages/chooseblock'
+  defaults:
+    _controller: '\Drupal\nidirect_landing_pages\Controller\LandingPagesChooseBlockController::build'
+    _title: 'build'
+  requirements:
+    _permission: 'access content'
```

Comment:
Does this mean anonymous users can also call this controller method? Can we set it to be something like 'edit landing pages' which editors are more likely to have?

Reply:
Doh, my bad